src/twisted/internet/_glibbase.py

```diff
@@ -126,14 +150,18 @@ def __init__(self, glib_module: Any, gtk_module: Any, useGtk: bool = False) -> N
         self._crash = _loopQuitter(self._glib.idle_add, self.loop.quit)
         self._run = self.loop.run
 
-    def _handleSignals(self):
+    def _reallyStartRunning(self):
         # First, install SIGINT and friends:
-        super()._handleSignals()
+        super()._reallyStartRunning()
         # Next, since certain versions of gtk will clobber our signal handler,
         # set all signal handlers again after the event loop has started to
-        # ensure they're *really* set. We don't call this twice so we don't
-        # leak file descriptors created in the SIGCHLD initialization:
-        self.callLater(0, posixbase.PosixReactorBase._handleSignals, self)
+        # ensure they're *really* set.
```

Comment:
Hm.  Yes, that would be good to know.  The test suite passes on my system with this whole method deleted.  I don't know if that means our test suite is insufficient or if this code is only for obsolete versions of glib or something else.

I'm reluctant to actually delete it since I don't know and since you didn't make this a blocker I'll refrain from doing so _or_ further investigating right here.

Reply:
https://github.com/twisted/twisted/issues/11762 and linked in the code